### ISSUE TITLE
A way of centering (or attempting to center) the whole graph in the viewframe

### ISSUE THREAD

**Issue by adamstraube** [anvaka/VivaGraphJS#185]
As per the heading I am trying to center a graph in the viewframe but have been unsuccessful.

I have checked through all examples and could not find  this functionality. Is there any way to do this currently?

**Comment by adamstraube** [comment]
Part answer is here (relating to Zooming out if graph gets too big) https://github.com/anvaka/VivaGraphJS/issues/90

Still looking for a way to scroll graph to the center of the viewframe.

**Comment by adamstraube** [comment]
A way to Pan the screen to a node: https://github.com/anvaka/VivaGraphJS/issues/63
This works for me as I have a node that is in the center most of the time.

The next problem I ran into is the graph is still finding its layout when I ran the above 2 snippets of code. 
I ended up putting the snippets of code in a function block and setting a timeout of 2 seconds eg.

```javascript
	var check = function(){
		adjustView(nodeId, layout); // Code to adjust the zoom and pan of view
	}
	setTimeout(check, 2000); // run check function in 2 seconds
```
